src/functions-and-arrays.js

```diff
@@ -77,8 +81,8 @@ function averageNumbers(numbersAvg) {
 };
```

Comment:
You could also have used the sum function previously declared instead of typing the sum logic once again. After all, the beauty of functions is that they can be reused! :smile:

Reply:
True, good point! Thanks :)